fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -705,7 +705,7 @@ public void execute() throws Exception {
         } catch (UserException e) {
             String sql = originStmt != null ? originStmt.originStmt : "";
             // analysis exception only print message, not print the stack
-            LOG.info("execute Exception, sql: {}, error: {}", sql, e.getMessage());
+            LOG.warn("execute Exception, sql: {}, error: {}", sql, e.getMessage());
```

Comment:
We only use warning for the errors which will cause system failure. Info for expected warnings

Reply:
ok, not sure if it is just StarRocks' log style. But it is strange to me 😄